### PR TITLE
Add claim rewards badge

### PR DESCRIPTION
**Pull request by yivlad**
Can be tested using https://impersonator.xyz/ and address `0xf8de75c7b95edb6f1e639751318f117663021cf0`

### CODE REVIEW

File: packages/app/src/features/navbar/logic/getRewardsInfo.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { MarketInfo } from '@/domain/market-info/marketInfo'
+import { UseQueryResult } from '@tanstack/react-query'
+import { RewardsInfo } from '../types'
+
+export function getRewardsInfo(marketInfo: UseQueryResult<MarketInfo>): RewardsInfo {
+  return {
+    isLoading: marketInfo.isLoading,
+    rewards:
+      marketInfo.data?.userRewards.map((reward) => ({
```
Comment (oskarvu) [inline packages/app/src/features/navbar/logic/getRewardsInfo.ts:9]: nit and optional: bit easier to read imo.

```
(marketInfo.data?.userRewards ?? []).map((reward) => ({
  token: reward.token,
  amount: reward.value,
})),
```

File: packages/app/src/features/navbar/components/rewards-badge/RewardsBadge.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import { Tooltip, TooltipContentLong, TooltipTrigger } from '@/ui/atoms/tooltip/Tooltip'
+import { NavbarActionWrapper } from '../NavbarActionWrapper'
+import { RewardsDetails } from './RewardsDetails'
+import { Reward } from './types'
+
+export interface RewardsProps {
+  rewards: Reward[]
+  isLoading: boolean
+}
+export function RewardsBadge({ rewards, isLoading }: RewardsProps) {
```
Comment (krzkaczor) [inline packages/app/src/features/navbar/components/rewards-badge/RewardsBadge.tsx:14]: IMO: since most of the users won't have any rewards and won't see the badge, i would tweak it to render null on loading

File: packages/app/src/features/navbar/components/airdrop-badge/AirdropDetails.tsx [hunk 3]
```diff
@@ -16,7 +16,7 @@ interface AirdropDetailsProps {
 
 export function AirdropDetails({ amount, precision, isLoading, isGrowing }: AirdropDetailsProps) {
   return (
-    <div className="flex flex-col text-basics-dark-grey text-xs">
+    <div className="flex w-[calc(100vw-48px)] flex-col text-basics-dark-grey text-xs lg:w-auto">
```
Comment (oskarvu) [inline packages/app/src/features/navbar/components/airdrop-badge/AirdropDetails.tsx:19]: Is this change needed? It doesn't seem to do anything when I checked locally.

File: packages/app/src/features/navbar/components/rewards-badge/RewardsBadge.tsx (hunk 2 above)
Comment (oskarvu) [reply packages/app/src/features/navbar/components/rewards-badge/RewardsBadge.tsx:14]: Should we maybe show rewards badge anyway with 0 value, for educational or encouragement purposes?

File: packages/app/src/features/navbar/components/airdrop-badge/AirdropDetails.tsx (hunk 3 above)
Comment (yivlad) [reply packages/app/src/features/navbar/components/airdrop-badge/AirdropDetails.tsx:19]: This improves the popup view on mobile

**Review by oskarvu — APPROVED** [review]

**Review by krzkaczor — APPROVED** [review]
